views/page/course/module/events/events-overview.html

```diff
@@ -113,8 +113,8 @@ <h3 class="govuk-heading-m">Course attendees</h3>
                             text: 'Edit time and date'
                         },
                         {
-                            link: '#',
-                            text: 'Edit location and capcity'
+                            link: '/content-management/courses/' + course.id + '/modules/' + module.id + '/events/' + event.id + '/location/',
```

Comment:
INFO - Does this section not take the expression bit for readability ? `link: \`content...{var}..\``

Reply:
Unfortunately this isn't possible because it's already in a `set` block rather than being called directly in the html. I can do something like this...
```
{% set editTimeAndDateLink %}/content-management/courses/{{course.id}}/modules/{{module.id}}/events/{{event.id}}/dateRanges/{% endset %}
                {% set actions = [
                        {
                            link: editTimeAndDateLink,
                            text: 'Edit time and date'
                        },
...
```
I'm not sure it's any more readable though. What do you think?

Reply:
Leave it as it is please then